static/coffee/modals.coffee

```diff
@@ -90,13 +90,19 @@ modals.controller 'NewCaseModalController', ['$scope', '$uibModalInstance', 'sum
 #=====================================================================
 # Assign to partner modal
 #=====================================================================
-modals.controller 'AssignModalController', ['$scope', '$uibModalInstance', 'title', 'prompt', 'partners', ($scope, $uibModalInstance, title, prompt, partners) ->
+modals.controller 'AssignModalController', ['$scope', '$uibModalInstance', 'title', 'prompt', 'partners', 'users', 'UserService', ($scope, $uibModalInstance, title, prompt, partners, users, UserService) ->
```

Comment:
Am fine with you guys replacing this with `ModalService.assign()` if that's preferable